packages/venia-ui/lib/components/Checkout/shippingAddressSummary.js

```diff
@@ -1,8 +1,16 @@
 import React, { Fragment } from 'react';
 import { array, bool, shape, string } from 'prop-types';
 
+import { useCheckoutContext } from '@magento/peregrine/lib/context/checkout';
+
+import isObjectEmpty from '../../util/isObjectEmpty';
```

Comment:
In another place we imported this from Peregrine - do we want to do that here too?